pages/tool_assembly/omero_assembly.md

```diff
@@ -0,0 +1,55 @@
+---
+title: OMERO
+contributors: [Jean-Marie Burel]
+tags: [data organisation, storage, data analysis]
+page_tag: ome
+summary: Tool assembly for managing imaging data
```

Comment:
```suggestion
summary: Tool assembly for managing microscopy imaging data
```

Reply:
OMERO supports many imaging data including jpeg, png. We have also worked with marine biologists taking for example images of seabeds or bird colonies. It is not limited to microscopy data, even if our initial focus was on microscope images.

Note that OMERO supports clinical data too.

Reply:
Valid point. Let's keep it like this then.